stream/src/placeholder/instance.rs

```diff
@@ -1,6 +1,9 @@
 use super::{
     handshake::{create_handshake, Handshake, IncomingHandshake},
-    utils::nonce_bytes,
+    utils::{
+        codec::{recv_frame, send_frame},
+        nonce::nonce_bytes,
```

Comment:
nit: change this `nonce::bytes` or something without the `nonce::nonce` stutter.